examples/google_adk/.gitignore

```diff
@@ -0,0 +1,8 @@
+vendor/
```

Comment:
I think most of these ignores have been included in global gitignore. Please check,